schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -462,6 +476,17 @@
         "InstallerType": {
           "$ref": "#/definitions/InstallerType"
         },
+        "NestedInstallers": {

```

Comment:
>NestedInstallers [](http://example.com/codeflow?start=9&length=16)

nit: a name that's specific to archive would be better to differentiate from the "nested" installer type concept in AppAndFeatures entries ones

Reply:
I changed the name to "ArchiveInstallers" as to not confuse it with the "nested" installerType concept in AppsAndFeatures